tutorials/workflow/java/workflow-management/README.md

```diff
@@ -0,0 +1,33 @@
+# Workflow Management
+
+This tutorial demonstrates the various APIs to manage a workflow instance, these methods include:
+
+- Scheduling a workflow instance
+- Getting the workflow instance state
+- Suspending the workflow instance
+- Resuming the workflow instance
+- Terminating the workflow instance
+- Purging the workflow instance
+
+For more information on workflow management, see the [Dapr docs](https://docs.dapr.io/developing-applications/building-blocks/workflow/howto-manage-workflow/).
+
+## Inspect the code
+
+Open the `Program.cs` file in the `tutorials/workflow/csharp/workflow-management/WorkflowManagement` folder. This file contains the endpoint definitions that use the workflow management API. The workflow that is being managed is named `NeverEndingWorkflow` and is a counter that will keep running once it's started.
```

Comment:
This is still the .NET version of the README

Reply:
done